tutorials/lab/coral2.rst

```diff
@@ -18,96 +18,82 @@ Things to Know
     Attempting to run further multi-node jobs will cause the excess jobs
     to fail. There is no limit on *submitted* multi-node jobs, and
     single-node jobs do not count towards the limit.
-#.  All nested Flux instances (e.g. instances created with ``flux batch``,
-    ``flux alloc``, or ``flux submit ... flux start``
-    should meet one of the following criteria:
+#.  All Flux instances should meet one of the following criteria:
 
     - Occupy a single node
     - Have exclusive access to the nodes they are running on (e.g. they
       do not share their resources with sibling instances).
 
     Instances that do not meet one of the above criteria will not work properly.
 
+By default Flux reserves ports 11000-11999 for itself. At any given
```

Comment:
I didn't know this! What are the ports used for?

Reply:
They're used by Cray PMI for communication, I believe.